### PR TITLE
Migrate to Elm 0.19.

### PR DESCRIPTION
**Pull request by n1k0**
```
$ git clone https://github.com/allo-media/elm-kitchen.git
$ cd elm-kitchen
$ git checkout elm-0.19
$ cd skeleton
$ npm install
$ npm start && open http://localhost:3000
```

TODO:

- [x] remove elm-css 0.19 branch source code once it's released
- [x] check deploy script

### CODE REVIEW

**Comment by n1k0** [comment]
Now waiting for an official Elm 0.19 compatible release of elm-css.

**Comment by n1k0** [comment]
Maybe @fbentz wants to have a look?

File: skeleton/src/Route.elm [hunk 1]
```diff
@@ -37,10 +39,13 @@ routeToString route =
...
     in
     "#/" ++ String.join "/" pieces
 
 
-modifyUrl : Route -> Cmd msg
-modifyUrl =
-    routeToString >> Navigation.modifyUrl
+modifyUrl : Nav.Key -> Route -> Cmd msg
+modifyUrl key route =
+    Nav.replaceUrl key (routeToString route)
```
Comment (fbentz) [inline skeleton/src/Route.elm]: Why not with pushUrl to have a history?

**Comment by n1k0** [comment]
Released v3.0.0.

**Pull request merged**